lib/project_types/theme/commands/pull.rb

```diff
@@ -34,10 +34,15 @@ class Pull < ShopifyCLI::Command::SubCommand
           flags[:ignores] ||= []
           flags[:ignores] |= pattern
         end
+        parser.on("-f", "--force") { flags[:force] = true }
       end
 
       def call(_args, name)
         root = root_value(options, name)
+        return unless exist_and_empty?(root) ||
```

Comment:
When the user answers with _No_, the exit code is `0`.
I think that's okay, as the program finished successfully.